libraries/datadogstatsd.php

```diff
@@ -239,28 +282,98 @@ public static function event($title, $vals = array()) {
             }
         }
 
-        $body = json_encode($vals); // Added in PHP 5.3.0
-        $opts = array(
-            'http'=> array(
-                'method' => 'POST',
-                'header' => 'Content-Type: application/json',
-                'content' => $body
-            )
-        );
-
-        $context = stream_context_create($opts);
+        /**
+         * @var boolean Flag for returning success
+         */
+        $success = true;
 
         // Get the url to POST to
         $url = self::$__datadogHost . self::$__eventUrl
              . '?api_key='          . self::$__apiKey
              . '&application_key='  . self::$__applicationKey;
 
-        // Send, suppressing and logging any http errors
+        $curl = curl_init($url);
```

Comment:
Anyway, `ext-curl` or another requirement like `guzzle` must be in composer.json.